User Engagement Lead/survey_questions.md

```diff
@@ -0,0 +1,79 @@
+# Survey questions
+
+## Readme
+
+This survey aims at evaluating users’ experiences with ESMValTool and
+discovering users’ needs.  It is meant for the users of ESMValTool in all levels
+of expertise. Some criteria to design the questions:
+
+- It does not take more than 10 (?) minutes
+- It does not have overlapping questions
+- It does not include questions with yes/no answers
+- It does not collect personal data, but it has some profiling questions
+- It covers different aspects of ESMValTool like application, usage, documentation and community
+
+The survey can be implemented in a survey app like surveymonkey that does not
+collect personal data. The owner is “user engagement team”. They will
+edit/run/evaluate the survey and communicate the results to other teams if
+needed.
+
+## Survey content
+
+Hi! We in [ESMValGroup](https://www.esmvaltool.org/), would love to hear about
+your research and experiences using ESMValTool. Here, there are a few questions
+which help to create your story. Your answers will help us to improve ESMValTool
+and hence enhance the science that you conduct. All responses are recorded
+anonymously. Let’s begin:
+
+**Describe your research/case study and challenges:**
+
+1. Can you describe your research/case study?
+2. Can you underline the challenges in your research/case study?
+3. How would you categorize your expertise level in using ESMValTool? (Beginner,
+   competent practitioner, expert)
+
+**Software:**
+
+ESMValTool provides many recipes (scientific workflows), features and functions to analyze data for Earth system
+models. See the [list of preprocessors](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/recipe/preprocessor.html#preprocessor) 
+and [list of recipes](https://docs.esmvaltool.org/en/latest/recipes/index.html).
+
+4. Which recipes/preprocessors in ESMValTool do you think are most useful for your
+research/case study?
+5. How satisfied are you with the software's:
+    - Ease of use (1-10)
+    - Development workflow (1-10)
+    - Reliability (1-10)
+
+**Documentation and tutorial:**
+
+ESMValTool [documentation](https://docs.esmvaltool.org/en/latest/) covers
+different aspects of research: How the code work, how to obtain and process
+data, and what are best practices in developing code. There is also an online
+[tutorial](https://esmvalgroup.github.io/ESMValTool_Tutorial/) that shows not
+only how to run an experiment but also how to develop your own code.
+If you are familiar with documentation and tutorial, please tell us:
+
+6. What are the most useful parts?
+7. What topics are missing that you want to learn about?
+8. How much do you agree with the following statements?
+    - It is easy to find relevant information from the documentation. (1-10)
+    - The instructions in the documentation and tutorial are complete and easy
+      to follow. (1-10)
```

Comment:
I would seperate the questions about documentation and tutorial. There could be different answers...